Extensions/3D/AmbientLight.ts

```diff
@@ -6,6 +6,9 @@ namespace gdjs {
         target: EffectsTarget,
         effectData: EffectData
       ): gdjs.PixiFiltersTools.Filter {
+        if (typeof THREE === 'undefined') {
```

Comment:
doing a `if (!THREE)` is not enough ?

Reply:
No because it would make an "undefined variable reference" exception.
It's used at other places in the code base, I only copied it.